numpy/core/src/multiarray/einsum.dispatch.c.src

```diff
@@ -1,47 +1,26 @@
 /*
- * This file provides optimized sum of product implementations used internally
- * by einsum.
+ * This file contains the implementation of the 'einsum' function,
+ * which provides an einstein-summation operation.
  *
  * Copyright (c) 2011 by Mark Wiebe (mwwiebe@gmail.com)
  * The University of British Columbia
  *
  * See LICENSE.txt for the license.
  */
+/**
+ * @targets $maxopt baseline
+ * SSE2 (AVX2 FMA3) AVX512F
+ * NEON NEON_VFPV4
+ * VSX VSX2
+ */
 
-#define NPY_NO_DEPRECATED_API NPY_API_VERSION
 #define _MULTIARRAYMODULE
-
-#include <numpy/npy_common.h>
-#include <numpy/ndarraytypes.h>  /* for NPY_NTYPES */
+#include <npy_pycompat.h>
```

Comment:
I suspect this isn't needed. This file doesn't touch the python API, does it?

Reply:
yes,  `npy_pycompat.h` is not necessary.